VogtPayroll5/Payroll.cs

```diff
@@ -1,34 +1,90 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Text;
+using System.IO;
 
 namespace VogtPayroll5
 {
     class Payroll
     {
+        //total number of hours worked by all employees, and total gross pay as well
+        private static int countHours;
```

Comment:
Dont use a static variable to perform a temporary calculation. That is a code smell. You can almost guarenteed calculate this without a static class variable.